documentation/asciidoc/computers/camera/external_trigger.adoc

```diff
@@ -0,0 +1,56 @@
+== External Trigger on the Global Shutter Camera
+The global shutter camera can be triggered externally by pulsing the XTR pin on the board.
+Multiple cameras can be connected to the same pulse leading to a better way to synchronise two cameras.
+
+The low puslewidth plus an extra 14.26us is equal to the exposure time. i.e. a low pulse of 10000us leads to an exposure time of 10014.26us.
+Framerate is directly controlled by how often you pulse the pin. A PWM frequency of 30Hz will lead to a framerate of 30 frames per second. 
+
+image::images/external_trigger.jpg[Image showing pulse format]
+
+=== Preparation
+
+First, remove/desolder resistor 22 on the GS Camera board. This connects GP0 to XTR and without removal, the camera will not operate
+in external trigger mode.
+
+Next solder a wire to the touchpoint of XTR on the GS Camera board. 
+Connect these to the pico - XTR to any pin. Pin 28 is used in this example. Ground is not needed to be connected.
```

Comment:
I think 'pico' needs to be capitalised as 'Pico'? Also, this is the first time that the word 'Pico' appears in this page, so it might need a bit of earlier explanatory context?